libs/common/src/types/portal/amendment.ts

```diff
@@ -1,4 +1,11 @@
 import z from 'zod';
-import { PORTAL_FACILITY_AMENDMENT_USER_VALUES } from '../../schemas';
+import { FacilityAmendment, Prettify } from '..';
+import { PORTAL_FACILITY_AMENDMENT, PORTAL_FACILITY_AMENDMENT_USER_VALUES } from '../../schemas';
 
 export type PortalFacilityAmendmentUserValues = z.infer<typeof PORTAL_FACILITY_AMENDMENT_USER_VALUES>;
+
+type PortalFacilityAmendment = z.infer<typeof PORTAL_FACILITY_AMENDMENT>;
```

Comment:
Hmmmmm, do we want to replace the existing PortalFacilityAmendment type with a z.infer? It's possible to do some fun stuff with `discriminatedUnion` that might help us out? 

I think leave for now & maybe refactor this down the line

Reply:
Should any of these be optional/nullable?

Reply:
Good spot, thank you!